tensorflow/core/public/version.h

```diff
@@ -26,7 +26,7 @@ limitations under the License.
 
 // TF_VERSION_SUFFIX is non-empty for pre-releases (e.g. "-alpha", "-alpha.1",
 // "-beta", "-rc", "-rc.1")
-#define TF_VERSION_SUFFIX ""
+#define TF_VERSION_SUFFIX "-device-annotation-mkl-support"
```

Comment:
This definitely shouldn't be changed.  If anything, we could potentially check for a previous definition

e.g.
```
#ifndef TF_VERSION_SUFFIX
#define TF_VERSION_SUFFIX ""
#endif
```

Reply:
sorry my bad. just need to suffix sth so I know what features that I added into my personal build. not intended to merge into the upstream... I already removed this

Reply:
Looks like this is still here, needs to be removed.